tests/casp/oci_install.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Start CaaSP installation
+# Maintainer: Martin Kravec <mkravec@suse.com>
+
+use strict;
+use warnings;
+use base "y2logsstep";
+use testapi;
+
+sub run() {
+    send_key $cmd{install};
+
+    assert_screen 'inst-userpasswdtoosimple';
+    send_key 'ret';
+
+    # We need to wait a bit for the disks to be formatted
+    assert_screen "inst-packageinstallationstarted", 120;
+
+    # Check details view
+    # Check release notes
```

Comment:
well, I merged in before. This is just a generic remark that TODOs in source are often not helpful because mainly the author knows what they are about and if you grep for TODO or FIXME in source code you will commonly find stuff that is there for ages. Better keep the TODO list in tickets